### ISSUE TITLE
Compressed CSS files are unusable

### ISSUE THREAD

**Issue by backplane-import** [sidewinderlabs/backplanejs#8]
### Imported from backplanejs Google Code [issue 8](http://code.google.com/p/backplanejs/issues/detail?id=8).

<table>
    <tr>
        <th>Reporter</th>
        <td><a href="https://github.com/markbirbeck">markbirbeck</a></td>
    </tr>
    <tr>
        <th>Date</th>
        <td>1 Dec 2009  4:12:21 PM UTC</td>
    </tr>
</table>

The compressed CSS files are unusable in all browsers.

A temporary workaround has been placed into the build process that simply 
overwrites the compressed file with the uncompressed one.

In other words, backplane-min.css is not actually compressed.

---

Owner set to ----

Ticket status set to **Accepted**

**Module**: Tools
**Priority**: Medium
**Type**: Defect

**Comment by backplane-import** [comment]
### Comment by [markbirbeck](https://github.com/markbirbeck) on 10 Jan 2010 10:15:35 PM UTC

Making this a low priority because there is a workaround in place.

---

**Updates**

**Priority**: Medium &#8594; Low

**Comment by backplane-import** [comment]
### Update by [markbirbeck](https://github.com/markbirbeck) on 10 Jan 2010 10:16:21 PM UTC

**Component**: Scripts

**Comment by backplane-import** [comment]
### Update by [markbirbeck](https://github.com/markbirbeck) on 28 Feb 2010 10:37:01 AM UTC

**Component**: Scripts &#8594; Tools
